### PR TITLE
Added BMI calculator project

### PR DESCRIPTION
**Pull request by rahul975337**
Please check my pr

### CODE REVIEW

**Comment by rahul975337** [comment]
please mention the hacktoberfest-accepted label so that this repository is eligible for hacktoberfest this would help me

File: bmicalc/android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#Fri Jun 23 08:50:38 CEST 2017
+distributionBase=GRADLE_USER_HOME
```
Comment (hiashutoshsingh) [inline bmicalc/android/gradle/wrapper/gradle-wrapper.properties:2]: what the ^M here ?

**Review by hiashutoshsingh — CHANGES_REQUESTED** [review]

File: bmicalc/android/gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (rahul975337) [reply bmicalc/android/gradle/wrapper/gradle-wrapper.properties:2]: don't know why it is coming but it is not showing in my editor I've pushed it once again can you please check

Comment (rahul975337) [reply bmicalc/android/gradle/wrapper/gradle-wrapper.properties:2]: I guess it won't create any problem

**Review by hiashutoshsingh — APPROVED** [review]

**Pull request merged**